docs/docs/60-new-docs/30-faqs.md

```diff
@@ -4,6 +4,179 @@ sidebar_label: FAQs
 
 # Frequently Asked Questions
 
-:::warning
-Kent will complete these by the end of the week.
-:::
+Welcome to the FAQ page. Here you'll find answers to some of the most common
+questions about Kargo.
+
+## General Questions
+
+### _What is Kargo?_
+
+Kargo is an unopinionated
+[continuous promotion](#what-exactly-is-continuous-promotion-anyway) platform
+that helps developers orchestrate the movement of new code and configuration
+through the various "stages" of their applications' lifecycles using GitOps
+principles.
+
+### _What exactly is "continuous promotion," anyway?_
+
+If you have some familiarity with GitOps, you know that _GitOps agents_ like
+[Argo CD](https://argoproj.github.io/cd/) excel at making the actual state of a
+Kubernetes cluster reflect _desired state_ that is expressed declaratively and
+stored in a Git repository.
+
+GitOps provides no specific guidance, however, on how desirable changes can be
+propagated from the desired state of one stage in an application's lifecycle to
+the desired state of the next. Any means by which such a thing might be
+accomplished, we would consider to be a "promotion process." When that process
+is codified into a pipeline and either partly or fully automated, we consider it
+"continuous promotion."
+
+**_Follow up question: Why is this not "continuous deployment?"_**
+
+Because the processes we're focused on do not _perform_ deployments. They focus
+on propagating changes to desired state so that a GitOps agent like Argo CD can
+then perform the heavy lifting.
+
+### _So “stage” is just another word for “environment?”_
+
+Not exactly, but you could think of it that way if you like.
+
+"Environment" can be an overloaded term to begin with. We didn't want to make
+things worse. Beyond that, we find more flexibility in the term "stage." _Stage
+can mean whatever you need it to._ It could be a cluster. It could be a
+particular instance of your entire application. It could be one of just several
+microservices comprising your application. Or it could be a group of related
+microservices.
```

Comment:
I feel like this is missing a "punch".

I.e. it could bring forward that a "stage" is essentially just a promotion target for a set of artifacts ("freight"), and that it's up to the user to assess what this entails for their specific promotion pipeline.

I also feel it maybe should touch into a "stage" being a (hierarchical) "station" along a pipeline.